.github/workflows/static-test-entry-point.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/sh
+
+echo "::group::Installing additional dependencies to docker container" >&2
```

Comment:
is there a reason not to add this to riotdocker?

Anyway, I'm fine with it. We can always add it to riotdocker in a follow up

Reply:
It's not per se part of the docker, but of the CI-specific entry point to the docker. With this we get a nice grouping of the (rather unrelated) installation process that hides it once we go into the tests proper. See https://github.com/RIOT-OS/RIOT/pull/15756/checks?check_run_id=1691321476#step:5:13 and the following lines.

Reply:
There is a reason, why I have put that script with the Github workflows, as it is workflow specific ;-).

Reply:
TBH, this looks more like a hack. I would add this dependency directly to riotdocker.

Reply:
Yes, this is the preferred way, but we know how long this can take... If that is in, we can gladly remove this again (or keep the entry point script if something like this is needed in the future again).

Reply:
As said offline: This PR does not need to be rushed :-)

Reply:
If you open a PR to riotdocker, for something like this, it can be quite fast.

Reply:
[Here](https://github.com/RIOT-OS/riotdocker/pull/126) we go.

Reply:
ACK'd :)